src/app/calculator/utilities/power-factor-correction/power-factor-correction.service.ts

```diff
@@ -1,5 +1,5 @@
 import { Injectable } from '@angular/core';
-import { PFMonthlyOutputs, PowerFactorCorrectionInputs, PowerFactorCorrectionOutputs } from './power-factor-correction.component';
+import { MonthyInputs, PFMonthlyOutputs, PowerFactorCorrectionInputs, PowerFactorCorrectionOutputs } from './power-factor-correction.component';
 
 @Injectable()
 export class PowerFactorCorrectionService {
```

Comment:
Another note that could be useful.. I would pull out all the "generate" functions that are just returning objects and move them into their own .ts file and just export that function and call it where it is needed. If you have any questions on that just let me know.